zerver/lib/bulk_create.py

```diff
@@ -65,9 +65,9 @@ def bulk_create_users(realm: Realm,
         recipients_by_user_id[recipient.type_id] = recipient
 
     subscriptions_to_create: List[Subscription] = []
-    for user_id in user_ids:
+    for user_id, is_active in user_ids_and_active_status:
```

Comment:
I would prefer to just loop over `profiles_to_create` here.  (See my other comment above.)